.jsdoc.js

```diff
@@ -43,7 +43,7 @@ module.exports = {
     copyright: 'Copyright 2020 Google LLC',
     includeDate: false,
     sourceFiles: false,
-    systemName: '@google-cloud/dialogflow',
+    systemName: 'cx',
```

Comment:
this isn't right, will need to dig into that.

Reply:
oh, we've just discussed it and here I am finally reviewing the PR :)  Please ping me when this is regenerated.